src/Compatibility/Microsoft.DotNet.PackageValidation/xlf/Resources.zh-Hant.xlf

```diff
@@ -12,11 +12,6 @@
         <target state="new">API compatibility errors between '{0}' (left) and '{1}' (right):</target>
         <note />
       </trans-unit>
-      <trans-unit id="AssemblyLoadWarning">
```

Comment:
Unrelated to this PR, but I would have expected all of these to be localized at this point.  Are we missing something WRT loc handoff @dsplaisted?